### DISCUSSION TITLE
Possible to place a page action in a widget?

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#3090] [Help] [answered]
Hi all,

I have created a blade component called "infobox" which is meant to gather some scoped slots to be present above different tables (with then different content); it simply serves the purpose to give more explanation on what data is displayed in the table.

Anyways, I am thinking about having a button put to that component that will act like the page action "Create new resource entity", e. g. "Create new blog post" - though it should be part of the infobox blade.

Is there a smart way to solve this?

Thank you, have a nice evening

**Comment by danharrin** [comment, marked as answer]
Unfortunately not, Actions can only be mounted to pages at the moment.

**Reply by vv-ac** [reply]
Thank you!
